### DISCUSSION TITLE
Roadmap for 0.101

### DISCUSSION THREAD

**Discussion by jahav** [ClosedXML/ClosedXML#1976] [Announcements]
This is a tentative roadmap for 0.101, depending on the mood and time, it includes issues with the milestone.

* Represent everything that can occur in c tag of the SheetData element in ClosedXML structures
* write SheetData to the output file directly through streaming instead of creating DOM fo reach row and letting OpenXmlPartWriter write the DOM.
* Finish autofilter redesign. Ensure compliance with Excel and good API
* Add wilcard object with methods to parse and evaluate wildcards (needed for autofilter and formula)
* Don't check if cell intersects merged range/table on each insert to a cell (takes up 50% of time). Instead add it to the flags of a cell. When inserting a value, only check the flag and if not set, don't check against merged cells (requires IStructuralListener to keep everything up to date)
* Update XLParser to 1.6.2
* Implement array formulae.
* Move writing of parts from large file to individual parts writers in ClosedXML.IO
* Make a fuzzer for checking of function implementation edge cases against excel
* `AdjustToContent` needs to be sped up

**Comment by MorgeMoensch** [comment]
Thanks for your work on this cool library!

Any estimate as to how soon this will be released (at least in beta)?

**Comment by jahav** [comment]
@MorgeMoensch I aim for the first half of the March. I will probably release beta/RC package a week or two before the release.

**Comment by jahav** [comment]
I had some issues in RL, release will be moved to the end of Match.

**Reply by MorgeMoensch** [reply]
No worries, I wish you all the best!

**Comment by rubeesh** [comment]
Any update on the next release?
